### ISSUE TITLE
Web interface ZBX_SERVER_HOST and ZBX_SERVER_PORT are missing by default

### ISSUE THREAD

**Issue by ypicard** [cetic/helm-zabbix#60]
**Describe the bug**
When deploying the chart for the first time with `zabbixweb` enabled, the dashboard complains about the zabbix server not being available on `localhost`.

It is because the default values of `ZBX_SERVER_HOST` and `ZBX_SERVER_PORT` (https://github.com/cetic/helm-zabbix/blob/master/templates/deployment-zabbix-web.yaml#L47) for the web service are not set in `values.yaml` for `zabbixweb` (https://github.com/cetic/helm-zabbix/blob/master/values.yaml#L231).

I have made it work by manually setting them to the following:
```
ZBX_SERVER_HOST: zabbix-zabbix-server
ZBX_SERVER_PORT: 10051
```

**Comment by aeciopires** [comment]
Hi @ypicard!

Thanks for report this issue.

This problem is solved using other approach in PR #54, that will not use this variables. Righ @sa-ChristianAnton?   

@ypicard, if you have some time, could you take a look at this PR?

We are close to finalizing the tests and adjustments to publish version 3.0.0 of the helm chart which changed very significant changes.

**Comment by ypicard** [comment]
Looks great. I'll wait for the release, thank you very much.
